lib/Github/Api/Authorizations.php

```diff
@@ -12,110 +12,87 @@ class Authorizations extends AbstractApi
 {
     /**
      * List all authorizations.
-     *
-     * @return array
      */
-    public function all()
+    public function all(): array
     {
         return $this->get('/authorizations');
     }
 
     /**
      * Show a single authorization.
-     *
-     * @param $clientId
-     *
-     * @return array
      */
-    public function show($clientId)
+    public function show($clientId): array
     {
-        return $this->get('/authorizations/'.rawurlencode($clientId));
+        return $this->get('/authorizations/'.rawurlencode((string) $clientId));
     }
 
     /**
      * Create an authorization.
      *
-     * @param array $params
      * @param null $OTPCode
      *
-     * @return array
+     * @return array|null
      */
     public function create(array $params, $OTPCode = null)
     {
-        $headers = null === $OTPCode ? array() : array('X-GitHub-OTP' => $OTPCode);
+        $headers = null === $OTPCode ? [] : ['X-GitHub-OTP' => $OTPCode];
 
         return $this->post('/authorizations', $params, $headers);
     }
 
     /**
      * Update an authorization.
      *
-     * @param $clientId
-     * @param array $params
-     *
-     * @return array
+     * @return array|null
      */
     public function update($clientId, array $params)
     {
-        return $this->patch('/authorizations/'.rawurlencode($clientId), $params);
+        return $this->patch('/authorizations/'.rawurlencode((string) $clientId), $params);
     }
 
     /**
      * Remove an authorization.
      *
-     * @param $clientId
-     *
-     * @return array
+     * @return array|null
      */
     public function remove($clientId)
     {
-        return $this->delete('/authorizations/'.rawurlencode($clientId));
+        return $this->delete('/authorizations/'.rawurlencode((string) $clientId));
     }
 
     /**
      * Check an authorization.
      *
-     * @param $clientId
-     * @param $token
-     *
-     * @return array
+     * @return array|null
      */
     public function check($clientId, $token)
     {
-        return $this->get('/applications/'.rawurlencode($clientId).'/tokens/'.rawurlencode($token));
+        return $this->get('/applications/'.rawurlencode((string) $clientId).'/tokens/'.rawurlencode($token));
     }
 
     /**
      * Reset an authorization.
      *
-     * @param $clientId
-     * @param $token
```

Comment:
We should not remove these with adding type annotations.

Reply:
What is the type here?

Reply:
I’m not sure without reading the code and docs.